### DISCUSSION TITLE
Unable to enable the push.disable-consistency-check flag

### DISCUSSION THREAD

**Discussion by vi4vikas** [prometheus/pushgateway#595] [Q&A] [answered]
I am deploying PGW v1.5.1 using the [helm chart v2.2.0](https://github.com/prometheus-community/helm-charts/tree/prometheus-pushgateway-2.2.0/charts/prometheus-pushgateway). I can see the flag set up to be false but as I put
```
  extraArgs:
   - --push.disable-consistency-check=true
```
in my value.yaml file, the pod breaks as it fails to create the container. But with other flags like:
```
   - --persistence.file=/data/pushgateway.data
   - --persistence.interval=10m
```
it shows no issues. I tried 
```
  extraArgs:
   - --no-push.disable-consistency-check
```
that makes the pod stable and PGW comes up fine but I cannot see the flag changes in UI

Am I missing something?

**Comment by beorn7** [comment, marked as answer]
Sorry, for the confusion. Too many negations here… :smile: 

So:

- Consistency checks are _enabled_ by default, that's why you see _push.disable-consistency-check → false_ in the UI.
- To _disable_ the checks, you just have to pass in `--push.disable-consistency-check`.
- If you don't want to rely on the default and explicitly enable the checks, you could pass in `--no-push.disable-consistency-check`.

**Comment by vi4vikas** [comment]
Thanks tried the following and it worked:
```
  extraArgs:
    - --push.disable-consistency-check
```

😄
